Phaethon/WebClient/Content/HoverElement.css

```diff
@@ -0,0 +1,39 @@
+﻿.container{
```

Comment:
why make new css file for this? why not use existing site.css, as itll be needed most likely everywhere

Reply:
Because I don't need site.css for all pages, but hover.css is required for everypage

Reply:
site css should be the file for all site, idk whats in there right now, i think its some auto generated stuff which i dont use

Reply:
yeah, but its not used in several views, so...